src/startup_checks.cpp

```diff
@@ -1,34 +1,49 @@
 /*
-* Unpublished Copyright (c) 2009-2018 AutonomouStuff, LLC, All Rights Reserved.
-*
-* This file is part of the PACMod ROS 1.0 driver which is released under the MIT license.
-* See file LICENSE included with this software or go to https://opensource.org/licenses/MIT for full license details.
-*/
+ * Unpublished Copyright (c) 2009-2021 AutonomouStuff, LLC, All Rights Reserved.
+ *
+ * This file is part of the PACMod ROS 1.0 driver which is released under the MIT license.
+ * See file LICENSE included with this software or go to https://opensource.org/licenses/MIT for full license details.
+ */
 /*
  * Series of startup checks for the pacmod game control node.
  */
 
-#include "pacmod_game_control/startup_checks.h"
+#include "pacmod_game_control/publish_control.h"
 
 #include <string>
 
+bool PublishControl::run_startup_checks_error()
+{
+  bool willExit = false;
+
+  ros::NodeHandle pnh("~");
+
+  // Run startup checks
+  willExit = check_steering_stick_left_right(pnh);
+  willExit = check_vehicle_type(pnh);
+  willExit = check_controller_type(pnh);
+  willExit = check_scale_values(pnh);
```

Comment:
Wouldn't the last `willExit` overwrite the result of previous results?

Reply:
Haha, nice catch! it totally would. I moved this function from the bottom of the file but missed this.